ui-admin/src/util/familyUtils.tsx

```diff
@@ -0,0 +1,23 @@
+import { Family } from '@juniper/ui-core'
+import { uniq } from 'lodash/fp'
+import { isEmpty } from 'lodash'
+
+/**
+ * Returns a string of family names, separated by commas and an 'and' before the last name.
+ * For example, a family with the members 'Alice Tester', 'Bob Reviewer', and 'Charlie Examiner'
+ * would return 'Tester, Reviewer and Examiner'.
+ */
+export const getFamilyNames = (family: Family) : string => {
```

Comment:
Normally, putting the data type of the function return in the name is redundant, but here I think it really helps clarify what's happening
```suggestion
export const getFamilyNameString = (family: Family) : string => {
```